sdk/python/kfp/v2/components/types/artifact_types.py

```diff
@@ -19,9 +19,9 @@
 import os
 from typing import Dict, Generic, List, Optional, Type, TypeVar, Union
 
-_GCS_LOCAL_MOUNT_PREFIX = '/gcs/'
-_MINIO_LOCAL_MOUNT_PREFIX = '/minio/'
-_S3_LOCAL_MOUNT_PREFIX = '/s3/'
+_GCS_LOCAL_MOUNT_PREFIX = '/tmp/gcs/'
```

Comment:
This needs to be `/gcs/` for [GCS Fuse](https://cloud.google.com/storage/docs/gcs-fuse) support on Vertex Pipelines.
The content you write to the local path `/gcs/my_project/bucket/blob` would be automatically synced to `gs://my_project/bucket/blob`.

Reply:
Do you have an alternate solution for being able to execute in non-root images?

Reply:
I think it is an implementation decision by Vertex Pipelines backend. Following that, we probably have multiple places in code where we rely on this conversation between local path and gs uri. 
Another example:
https://github.com/kubeflow/pipelines/blob/3a621fb577756171c726a423ce68a513b9053b8f/v2/component/launcher.go#L655

Reply:
This PR does also change those.

But you're probably right that it's actually probably a problem closer to how Vertex use `gcsfuse` and the file permissions which are set there in my case.

However, do you not think that trying to create these directories on at the root of the filesystem is an unnecessary hurdle in the way of non-root containers?